### ISSUE TITLE
Hey is it possible to add bluetooth connectivity to this keyboard?

### ISSUE THREAD

**Issue by TimotejVarga** [coseyfannitutti/mysterium#32]
Hey is it possible to add Bluetooth connectivity to this keyboard?

**Comment by AsrieltheGoat** [comment]
Yes, but you need to modify the pcb by adding a bluetooth hid like HC-06 with a custom firmware (https://www.youtube.com/watch?v=y8PcNbAA6AQ) and add a power circuit like a power bank
